config/crd/bases/kueue.x-k8s.io_workloads.yaml

```diff
@@ -7437,8 +7437,6 @@ spec:
                 description: queueName is the name of the queue the Workload is associated
                   with. queueName cannot be changed once set.
                 type: string
-            required:
```

Comment:
Is this intended? :-)

Reply:
yes. Empty string is valid.

Reply:
But is empty string the same as unspecified string? Because that's what you have now.

Reply:
yes, the default is empty and empty is valid

Reply:
it's not great then but I guess it works.